CHANGELOG.rst

```diff
@@ -29,12 +29,16 @@ Internal Changes
 - **Helm chart**: Set the notebooks replica count to zero because it is replaced by the data services.
 - **Helm chart**: Add RBAC rules for Shipwright and Tekton resources in the Kubernetes cache.
 
+**Bug Fixes**
+
+- **Data services**: Better validation for URLs of Git repositories.
 
 Individual Components
 ~~~~~~~~~~~~~~~~~~~~~
 
 - `renku-data-services 0.35.0 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.35.0>`_
 - `renku-data-services 0.35.1 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.35.1>`_
+- `renku-data-services 0.35.2 <https://github.com/SwissDataScienceCenter/renku-data-services/releases/tag/v0.35.2>`_
```

Comment:
If we include this without the corresponding UI change, it will not be nice for users.